### PR TITLE
Feature: App-1437

### PR DESCRIPTION
**Pull request by sepehr2github**
## Description

- New MV parameters added to governance card in dao creation review page
- Fix the token required for proposal creation limit

Task: [App-1437](https://aragonassociation.atlassian.net/browse/APP-1437)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/containers/goLive/community.tsx [hunk 1]
```diff
@@ -103,6 +105,17 @@ const Community: React.FC = () => {
...
               </Dl>
+              <Dl>
+                <Dt>{t('labels.proposalCreation')}</Dt>
+                <Dd>
+                  {eligibilityType === 'token'
+                    ? t('createDAO.review.proposalCreation', {
+                        token: eligibilityTokenAmount,
+                        symbol: tokenSymbol,
+                      })
+                    : 'Anyone'}
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/goLive/community.tsx:116]: i18n :p

Comment (WissenIstNacht) [inline packages/web-app/src/containers/goLive/community.tsx]: i18n :p

File: packages/web-app/src/containers/goLive/governance.tsx [hunk 2]
```diff
@@ -42,31 +42,20 @@ const Governance: React.FC = () => {
...
-            </Dl>
-          )}
           <Dl>
-            <Dt>{t('labels.minimumApproval')}</Dt>
-            <Dd>{parseInt(minimumApproval)}%</Dd>
+            <Dt>{t('labels.supportThreshold')}</Dt>
+            <Dd>&gt;{parseInt(minimumApproval)}%</Dd>
+          </Dl>
+          <Dl>
+            <Dt>{t('labels.minimumParticipation')}</Dt>
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/goLive/governance.tsx:50]: This slightly changed (additional GTE sign). Reference: https://github.com/aragon/apps/blob/7fd13ae7bab8a02e8cd04b6f51a413ced6b00e68/packages/web-app/src/pages/settings.tsx#L224

File: packages/web-app/src/pages/createDAO.tsx [hunk 3]
```diff
@@ -168,8 +166,7 @@ const CreateDAO: React.FC = () => {
           ) ||
           errors.wallets ||
           errors.eligibilityTokenAmount ||
-          tokenTotalSupply === 0 ||
-          !(eligibilityTokenAmount !== 0)
+          tokenTotalSupply === 0
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/createDAO.tsx:166]: Hey, are those the fixes for what we talked about in the call this morning? If so, it's not quite what I expected/understood.

**Review by WissenIstNacht — CHANGES_REQUESTED** [review]
Thanks a lot! :)

File: packages/web-app/src/pages/createDAO.tsx (hunk 3 above)
Comment (sepehr2github) [reply packages/web-app/src/pages/createDAO.tsx:166]: if we always set a default value for eligibilityTokenAmount we don't need to force user touch the value all the time! That's why those conditions removed!

**Review by WissenIstNacht — APPROVED** [review]
Sorry about the comment on the mininum support threshold not working. Something must have been wrong with my localhost. I tested it again and it works just like we discussed 👌🏼

**Pull request merged**